spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe AdminUser do
+  it 'return type should be User' do
+    testuser = AdminUser.find_or_create
```

Comment:
Instead of calling this in each method, pull it out into a [`let` statement](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let) inside the describe block for this method.